extension/radia.js

```diff
@@ -0,0 +1,97 @@
+const axios = require('axios').default;
+
+async function listen(nodecg) {
+	const casters = nodecg.Replicant('casters');
+	const radiaSettings = nodecg.Replicant('radiaSettings');
+	var APIURL;
+	var Authentication;
+
+	if (
+		!nodecg.bundleConfig ||
+		typeof nodecg.bundleConfig.radia === 'undefined'
+	) {
+		nodecg.log.error(
+			`"radia" is not defined in cfg/${nodecg.bundleName}.json! The ability to import data via the Radia
+			Production API is not possible`
+		);
+		radiaSettings.value.enabled = false;
+		return;  // Exit from extension as keys aren't available to use.
+	} else {
+		APIURL = nodecg.bundleConfig.radia.url;
+		Authentication = nodecg.bundleConfig.radia.Authentication;
+		radiaSettings.value.enabled = true;
+	}
+
+	nodecg.listenFor('getLiveCommentators', async (data, ack) => {
+		getLiveCasters(APIURL, Authentication, radiaSettings.value.guildID)
+			.then((data) => {
+				// Clear all currently stored caster data
+				for (const [key, value] of Object.entries(casters.value)){
+					delete casters.value[key];
```

Comment:
`casters.value = {};` would probably work just as well.
Or, alternatively, overwrite the value with an already prepared value instead of writing into the replicant multiple times in a loop:
```js
const newCasters = {}
// something
casters[ID] = {data}
// something
casters.value = newCasters
```